identity/src/main/java/com/android/identity/IdentityCredentialStore.java

```diff
@@ -142,25 +144,61 @@ public abstract class IdentityCredentialStore {
     }
 
     /**
-     * Gets the default {@link IdentityCredentialStore} instance. This does not guarantee any
-     * particular {@link IdentityCredentialStoreCapabilities}, in particular it may be hardware-
-     * or software backed. The kind of instance returned by this method may change over time even
-     * on the same device, for example a device might gain the capability of a hardware-backed
-     * instance through an Android version upgrade.
+     * Get the default {@link IdentityCredentialStore} instance with no regard for featureVersion,
+     * as if implemented as {@code getDefaultInstance(context, FEATURE_VERSION_BASE)}.
      *
      * @param context the application context.
      * @return the {@link IdentityCredentialStore}.
      */
     public static @NonNull IdentityCredentialStore getDefaultInstance(@NonNull Context context) {
```

Comment:
What do you think, should I delete this method (introduced just yesterday) and update the old getInstance() method / its deprecation comment to say to call getDefaultInstance(context, FEATURE_VERSION_BASE)?

Generally the question is whether we should force callers to specify a featureVersion. I suspect yes, since not specifying one (before the new API existed) has led to bugs in 2/2 app implementations, so it's probably good to force callers to think about this? WDYT?